in_toto/util.py

```diff
@@ -136,3 +138,27 @@ def prompt_generate_and_write_rsa_keypair(filepath, bits):
   generate_and_write_rsa_keypair"""
   password = prompt_password()
   generate_and_write_rsa_keypair(filepath, bits, password)
+
+def color_code(msg, lvl, override=False):
```

Comment:
Maybe `color_encode` would be a better name? Also `level` seems short enough for not needing to shorten it.

Reply:
I think the module `in_toto.log` is a better place for this function. What do you think?

Reply:
Yup, renamed the function to `color_encode` and it's now sitting in `in_toto.log` as a function (and removed from util).